book.toml

```diff
@@ -8,6 +8,7 @@ command = "mdbook-open-on-gh"
 renderer = ["html"]
 
 [output.linkcheck]
+optional = true
```

Comment:
reason?

Reply:
People need to install mdbook-linkcheck first, remove optional and add it to readme